static/menu.js

```diff
@@ -0,0 +1,18 @@
+const navigationBar = $("#navbar");
+
+$("#contentBox").css("top", $("#navbar").css("height")); //Sets the content below the navbar.
+
+document.onscroll = () => {
```

Comment:
Ah, you talking about the `() => {}` part? I thought you where talking about line 3.